pkg/app/instances/docker.go

```diff
@@ -50,21 +58,76 @@ func (m *DockerInstanceManager) CreateHost(zone string, _ *apiv1.CreateHostReque
 	if zone != "local" {
 		return nil, fmt.Errorf("Invalid zone. It should be 'local'.")
 	}
-	return nil, fmt.Errorf("%T#CreateHost is not implemented", *m)
+
```

Comment:
nit: remove blank lines. These methods implementations are short ( which is good ) adding blank lines in between `if` blocks makes it look larger than it is.

Reply:
Remove blank lines inside a single function.